package.json

```diff
@@ -27,19 +24,24 @@
     "@types/node": "^20.11.16",
     "@wordpress/e2e-test-utils-playwright": "^0.19.0",
     "@wordpress/env": "^9.2.0",
+    "@wordpress/scripts": "^27.3.0",
     "babel-jest": "^29.7.0",
     "chalk": "^5.3.0",
+    "clsx": "^2.0.0",
     "concurrently": "^8.2.2",
+    "docker-compose": "^0.24.6",
```

Comment:
Since this is being added to devDependencies (which I agree with), we can likely remove the one in dependencies.